### PR TITLE
update Create ARM template for Session Host Configuration

### PR DESCRIPTION
**Pull request by philipktlin**
update Create ARM template for Session Host Configuration

### CODE REVIEW

File: ARM-wvd-templates/CreateAndProvisionHostPool/CreateHostpoolTemplate.json [hunk 1]
```diff
@@ -703,23 +709,18 @@
           "properties": {
             "vMSizeId": "[parameters('vmSize')]",
             "diskType": "[parameters('vmDiskType')]",
-            "vmCustomConfigurationUri": "[if(empty(parameters('customConfigurationTemplateUrl')), '', parameters('customConfigurationTemplateUrl'))]",
-            "vmCustomConfigurationParameterUri": "[if(empty(parameters('customConfigurationParameterUrl')), '', parameters('customConfigurationParameterUrl'))]",
+            "vmCustomConfigurationUri": "[if(empty(parameters('customConfigurationTemplateUrl')), null(), parameters('customConfigurationTemplateUrl'))]",
```
Comment (shanberg1) [inline ARM-wvd-templates/CreateAndProvisionHostPool/CreateHostpoolTemplate.json:712]: why this change?

Comment (philipktlin) [reply ARM-wvd-templates/CreateAndProvisionHostPool/CreateHostpoolTemplate.json:712]: Just want to do it right to set unused value to null in host pool creation scenario.  Although in Add VM scenario, I will update unused properties to empty string for PATCH api call.
